addons/api/app/transforms/number.js

```diff
@@ -0,0 +1 @@
+export { NumberTransform as default } from '@ember-data/serializer/transform';
```

Comment:
The boolean, date, number and string transforms are no longer auto exported for us. So we have to explicitly export.

Reply:
These were def. deprecation warnings. Below is an example:

![Pasted Graphic 6](https://github.com/user-attachments/assets/8f660dc1-1783-4dfd-954a-5ebb53804270)